src/services/services.json

```diff
@@ -1437,43 +1439,402 @@
+            {
+                "path": "/Ac/PvOnOutput/L3/Power",
+                "type": "float",
+                "name": "PV - AC-coupled on output L3 (W)"
+            },
+            {
+                "path": "/Ac/PvOnOutput/NumberOfPhases",
+                "type": "integer",
+                "name": "PV - AC-coupled on output Number Of Phases"
+            },
+            {
+                "path": "/Dc/Battery/ConsumedAmphours",
+                "type": "float",
+                "name": "Battery Consumed Amphours (Ah)"
+            },
+            {
+                "path": "/Dc/Battery/Current",
+                "type": "float",
+                "name": "Current (A)"
+            },
+            {
+                "path": "/Dc/Battery/Power",
+                "type": "float",
+                "name": "Battery Power (W)"
+            },
+            {
+                "path": "/Dc/Battery/Soc",
+                "type": "float",
+                "name": "Battery State of Charge (%)"
+            },
+            {
+                "path": "/Dc/Battery/State",
+                "type": "enum",
+                "name": "Battery state",
+                "enum": {
+                    "0": "idle",
+                    "1": "charging",
+                    "2": "discharging"
+                }
+            },
+            {
+                "path": "/Dc/Battery/TimeToGo",
+                "type": "float",
+                "name": "Battery Time to Go (h)"
+            },
+            {
+                "path": "/Dc/Battery/Voltage",
+                "type": "float",
+                "name": "Voltage (V)"
+            },
+            {
+                "path": "/Dc/Charger/Power",
+                "type": "integer",
+                "name": "AC-Chargers - power (W)"
+            },
             {
                 "path": "/Dc/Pv/Power",
                 "type": "float",
-                "name": "System Power"
+                "name": "MPPTs - power (W)"
             },
             {
                 "path": "/Dc/Pv/Current",
                 "type": "float",
-                "name": "System Current"
+                "name": "MPPTs - current (A)"
+            },
+            {
+                "path": "/Dc/System/Power",
+                "type": "float",
+                "name": "DC System (W)"
+            },
+            {
+                "path": "/Dc/Vebus/Current",
+                "type": "float",
+                "name": "VE.Bus charge current (A)"
+            },
+            {
+                "path": "/Dc/Vebus/Power",
+                "type": "float",
+                "name": "VE.Bus charge power (W)"
+            },
+            {
+                "path": "/Buzzer/State",
+                "type": "float",
+                "name": "Buzzer State",
+                "enum": {
+                    "0": "Off",
+                    "1": "On"
+                }
+            },
+            {
+                "path": "/SystemState/State",
```

Comment:
We don't want to add /SystemState here, as those are not exposed via modbus_tcp.